packages/hapi-auth-udaru-16/README.md

```diff
@@ -13,14 +13,14 @@ See the Udaru [website](https://nearform.github.io/udaru/) for complete document
 To install via npm:
 
 ```
-npm install @nearform/udaru-hapi-plugin
+npm install @nearform/hapi-auth-udaru-16
 ```
 
 ## Usage
 
 ```js
 const Hapi = require('hapi')
-const UdaruPlugin = require('@nearform/udaru-hapi-plugin')
+const UdaruPlugin = require('@nearform/hapi-auth-udaru-16')
```

Comment:
with this naming convention are we causing confusion, could someone think it is udaru-16 and not to do with hapi-16?

would it be better to have udaru-auth-hapi-16 & udaru-auth-hapi-17 for example and any other server support use the same convention udaru-auth-fastify etc.

Reply:
You're probably right.
The reason I don't want to use 17 in the newer package is because if in the future hapi goes 18 and we're still compatible we probably don't want to rename the package.

Reply:
yes I much preferred the old naming convention, I know it's not hapi convention but I don't care. Am leaning back towards: 

* udaru-core	
* udaru-hapi-16-plugin
* udaru-hapi-plugin
* udaru-hapi-server

Thoughts?

Reply:
yes sounds good to me.